src/opm/parser/eclipse/share/keywords/000_Eclipse100/W/WFOAM

```diff
@@ -0,0 +1,8 @@
+{
+  "name" : "WFOAM",
+  "sections" : [ "SPECIAL", "SCHEDULE" ],
```

Comment:
The `SPECIAL` section is a random 3+ years old error which is copy & pasted from PR to PR. Just remove it.

Reply:
Will do.